### DISCUSSION TITLE
default config file location

### DISCUSSION THREAD

**Discussion by Maabur** [Ulm-IQO/qudi-core#16] [Q&A] [answered]
Hello,
When I install just the new core 2021. Then install the new core 2021 with modules on the same environment it always looks for the config file that was located in the first installation (the core only). Is this normal ? Is there a way to change the default location of the config file ?
Cheers

**Comment by Neverhorst** [comment, marked as answer]
Everytime you load a new config file (via the file dialogue in the manager window), this file location will become the new default location. This is how it always has been... also in the old core.

Changing the default config file just via installation of an addon package (e.g. `qudi-iqo-modules`) is currently not possible. Things like that can only happen if we develop a proper way to _deploy_ qudi. Just _distributing_ a Python package via pip can not do such changes.

**Reply by Maabur** [reply]
ok thanks
